### PR TITLE
Fixed broken CI and some minor bug fixes for ecommerce credentials due to test credential overwrite

### PR DESCRIPTION
**Pull request by Therecanbeonlyone1969**
- Fixed CI for CommercialInvoice caused by improperly PriceSpeficiation
- Updated other credentials because chargeSpecification schema was removed and switched to PriceSpecification (reused chargeSpecification schema and tests)
- Fixed a naming bug for credential tests for 3 ecommerce credentials
- removed a couple of warnings because of the PriceSpecification issue
- no more warnings for any e-commerce schemas and their tests

### CODE REVIEW

**Comment by Therecanbeonlyone1969** [comment]
CI & CD passed .... https://github.com/cstoecker/traceability-vocab/actions/runs/616336361 ... https://github.com/cstoecker/traceability-vocab/actions/runs/616336359

@OR13 @mprorock @mkhraisha

**Comment by mprorock** [comment]
@OR13 merge order between this and #127 ?

**Comment by mprorock** [comment]
@Therecanbeonlyone1969 will approve once we sort merge order on the CD/CI refactor

**Comment by OR13** [comment]
both these PRs are big, so it will suck either way.

There are 2 ways to think of merge order.

- Smaller PRs first.

- First to passing CI test wins.

If we merge this first, I will likely need to reimplement it in my PR.

I can probably handle the merge in my PR.

**Review by OR13 — CHANGES_REQUESTED** [review]
@Therecanbeonlyone1969 can you remove the auto generated changes from this PR? I will then reimplement the test / code changes you have made in https://github.com/w3c-ccg/traceability-vocab/pull/127

Just overwrite `./docs/`  (except for schemas folder) with origin main docs and I will handle the merge in my PR

**Comment by Therecanbeonlyone1969** [comment]
> @Therecanbeonlyone1969 can you remove the auto generated changes from this PR? I will then reimplement the test / code changes you have made in #127
> 
> Just overwrite `./docs/` (except for schemas folder) with origin main docs and I will handle the merge in my PR

@OR13 ... ok will do that. Gimmer a hot minute.

**Comment by Therecanbeonlyone1969** [comment]
@OR13 ... done

File: docs/schemas/EcommerceWayBillTotals.json
Comment (OR13) [inline docs/schemas/EcommerceWayBillTotals.json:58]: ✅

File: docs/schemas/IssuerAgent.json
Comment (OR13) [inline docs/schemas/IssuerAgent.json:62]: ✅

File: docs/schemas/IssuerAgent.json
Comment (OR13) [inline docs/schemas/IssuerAgent.json:67]: ✅

File: docs/schemas/IssuerAgent.json
Comment (OR13) [inline docs/schemas/IssuerAgent.json:81]: ✅

File: docs/schemas/IssuerAgent.json
Comment (OR13) [inline docs/schemas/IssuerAgent.json:95]: ✅

File: docs/schemas/PriceSpecification.json
Comment (OR13) [inline docs/schemas/PriceSpecification.json:26]: ✅

File: docs/schemas/ShippingStop.json
Comment (OR13) [inline docs/schemas/ShippingStop.json:11]: ✅

File: packages/generate-examples/src/__fixture__/invoice.json
Comment (OR13) [inline packages/generate-examples/src/__fixture__/invoice.json:68]: ✅

File: packages/generate-examples/src/__fixture__/invoice.json
Comment (OR13) [inline packages/generate-examples/src/__fixture__/invoice.json:86]: ✅

File: packages/generate-examples/src/__fixture__/invoice.json
Comment (OR13) [inline packages/generate-examples/src/__fixture__/invoice.json:104]: ✅

File: packages/generate-examples/src/__tests__/EcommerceBindingDataRegistrationCredential.test.js
Comment (OR13) [inline packages/generate-examples/src/__tests__/EcommerceBindingDataRegistrationCredential.test.js:59]: ✅

File: packages/generate-examples/src/__tests__/EcommerceProductRegistrationCredential.test.js
Comment (OR13) [inline packages/generate-examples/src/__tests__/EcommerceProductRegistrationCredential.test.js:27]: ✅

File: packages/generate-examples/src/__tests__/EcommerceWayBillRegistrationCredential.test.js
Comment (OR13) [inline packages/generate-examples/src/__tests__/EcommerceWayBillRegistrationCredential.test.js:60]: ✅

File: packages/traceability-vocab/src/generators/EcommerceBindingDataRegistrationCredential.js
Comment (OR13) [inline packages/traceability-vocab/src/generators/EcommerceBindingDataRegistrationCredential.js:7]: ✅

File: packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js
Comment (OR13) [inline packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js:2]: ✅

File: packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js
Comment (OR13) [inline packages/traceability-vocab/src/generators/EcommerceWayBillTotals.js:16]: ✅

File: packages/traceability-vocab/src/generators/IssuerAgent.js
Comment (OR13) [inline packages/traceability-vocab/src/generators/IssuerAgent.js:3]: ✅

File: packages/traceability-vocab/src/generators/IssuerAgent.js
Comment (OR13) [inline packages/traceability-vocab/src/generators/IssuerAgent.js:35]: ✅

**Comment by OR13** [comment]
I have caught up with all these changes and am closing this PR.

**Pull request closed without merging**
